etna/lib/etna/test_auth.rb

```diff
@@ -74,7 +74,7 @@ def approve_hmac(request)
   end
   class TestHmac < Hmac
     def valid?
-      @test_signature == 'valid'
+      @test_signature == 'valid' || super
```

Comment:
This lets the `Etna::TestAuth` middleware also validate that "real" HMAC signatures are valid. The scenario happens when a Metis-signed HMAC URL is used in a test -- i.e. when POSTing to the `upload_url` returned by `authorize_upload`. Without this change, `valid?` will always return `false` in such a scenario. This change lets `TestAuth` handle test HMAC signatures as well as "real" ones.